addressing.mss

```diff
@@ -36,7 +36,10 @@
   [zoom >= 14][way_pixels > 3000],
   [zoom >= 17] {
     text-name: "[name]";
-    text-size: 11;
+    text-size: 8;
```

Comment:
don't use text this small, it poses readability problems, particularly with CJK glyphs (e.g. Japanese)